scripts/generator/graphql_generator/csharp/UI/NativePayButtonUI.cs.erb

```diff
@@ -0,0 +1,101 @@
+namespace <%= namespace %>.UI {
+    using UnityEngine;
+    using UnityEngine.UI;
+
+    using System.Runtime.InteropServices;
+    using System.Text;
+    using System;
+
+    /// <summary>
+    /// The Apple Pay payment button style C# enum. See https://developer.apple.com/documentation/passkit/pkpaymentbuttonstyle
+    /// for all available options.
+    /// </summary>
+    public enum ApplePayButtonStyle {
+        white,
```

Comment:
The convention is to use `Caps` case for enums in `C#`, Up to you 🤷‍♂️, but I think the rest of the Unity SDK is using this convention